### DISCUSSION TITLE
Column for Notes ignores white space at the beginning of its field

### DISCUSSION THREAD

**Discussion by KodeXall** [keepassxreboot/keepassxc#9298] [Ideas]
Rarely in the **Notes** field I use a description like _" Login in is done by using service XXX"_, where that service _XXX_ allows me to get into service _YYY_ for which such note I had added. And I also use field **Notes** for storing other important info

Unfortunately if I want to quickly single out such rare case after pressing on the header of this column an alphabetical sorting does occur - but instead of seeing every _" Login in is done by [...]"_ at the very top or bottom of now sorted list they are in its middle, because the leading space is ignored and they are place between entries beginning with **K** and **M**

And this omission of white space does not occur in case of the **Title** column, thus at best this is only inconsistency of rules or at worst an ordinary bug

**Comment by droidmonkey** [comment]
The notes column is really not useful. That is why we have the preview at the bottom.

**Comment by KodeXall** [comment]
If it will keep ignoring white space at its beginning it surely will continue being even less useful than it easily could become more useful
